geomesa-accumulo/geomesa-accumulo-distributed-runtime/src/main/resources/install-geomesa-namespace.sh

```diff
@@ -0,0 +1,105 @@
+#! /usr/bin/env bash
+#
+# Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0 which
+# accompanies this distribution and is available at
+# http://www.opensource.org/licenses/apache2.0.php.
+#
+
+# Installs a GeoMesa distributed runtime JAR into an Accumulo namespace
+
+while getopts ":u:p:n:g:h:d:" opt; do
+  case $opt in
+    u)
+      ACCUMULO_USER=$OPTARG
+      ;;
+    p)
+      ACCUMULO_PASSWORD=$OPTARG
+      ;;
+    n)
+      ACCUMULO_NAMESPACE=$OPTARG
+      ;;
+    g)
+      GEOMESA_JAR=$OPTARG
+      ;;
+    d)
+      NAMESPACE_DIR=$OPTARG
+      ;;
+    h)
+      HDFS_URI=$OPTARG
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      exit 1
+      ;;
+    :)
+      echo "Option -$OPTARG requires an argument." >&2
+      exit 1
+      ;;
+  esac
+done
+
+if [[ -z "$ACCUMULO_USER" ]]; then
+    echo "Accumulo username parameter is required: -u" >&2
+    ERROR=1
+fi
+
+if [[ -z "$ACCUMULO_NAMESPACE" ]]; then
+    echo "Accumulo namespace parameter is required: -n" >&2
+    ERROR=1
+fi
+
+if [[ -z "$GEOMESA_JAR" ]]; then
+    # cd to directory of script and look for GeoMesa JAR
+    cd "$( dirname "${BASH_SOURCE[0]}" )"
+    GEOMESA_JAR=$(ls | grep geomesa-accumulo-distributed-runtime)
+    if [ -z "$GEOMESA_JAR" ]; then
+        echo "Null GeoMesa JAR parameter encountered and could not find a GeoMesa distributed runtime JAR"
+        ERROR=1
+    else
+        echo "Null GeoMesa JAR parameter encountered, using $GEOMESA_JAR"
+    fi
+fi
+
+if [[ -z "$NAMESPACE_DIR" ]]; then
+    NAMESPACE_DIR="/accumulo/classpath"
+    echo "Null namespace directory parameter encountered, using $NAMESPACE_DIR"
+fi
+
+if [[ -z "$HDFS_URI" ]]; then
+    HDFS_URI=`hdfs getconf -confKey fs.defaultFS`
+    echo "Null HDFS URI parameter encountered, using $HDFS_URI"
+fi
+
+if [[ -z $ERROR && -z "$ACCUMULO_PASSWORD" ]]; then
+    read -s -p "Enter Accumulo Password: " ACCUMULO_PASSWORD
```

Comment:
would be nice to include the user here - "Please enter Accumulo password for user $ACCUMULO_USER: "